jordan-rps.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+  <div style="display:flex">
```

Comment:
Why include this style info in the HTML and not a CSS file? Just because the styles are so simple?

Reply:
@Meeploo yeah, and for the sake of the example I didn't feel like making a new css file haha